src/Digdir.Domain.Dialogporten.GraphQL/appsettings.prod.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
Burde vel helst kalt denne Production for å følge standarden 🤔  Hva tycks? IsProduction() kan vel ikke brukes. EnvironmentName == "prod" er det isåfall som må brukes AFAIK

Reply:
@MagnusSandgren  @elsand  hva tenker dere?

Reply:
Du har rett i at vi ikke kan benytte `IsProduction()` slik oppsettet er nå @arealmaas. Jeg ser for meg to måter å addressere denne utfordringen på:
1. Gjennom IaC ved å legge på en oversettelse fra "prod" til "Production" der det trengs. Blant annet når vi definerer dotnet environment variabelen.
2. Gjennom kode ved å lage vår egen [environment abstraksjon](https://stackoverflow.com/a/40721285) dersom vi noen gang trenger å gjøre noe ekstraordinært i produksjon. Meg bekjent trenger vi ikke det enda 🤔

Reply:
Gå for environment-abstraksjon in that case maybe? Vi bruker test, staging og prod for miljø, og test og staging har vel heller ikke noe helper-funksjon i Environment 🤔 Stemmer for å holde det til test, staging og prod da ja kanskje..

Reply:
Vi får gjøre noe slikt om vi ser at vi får behov.